sequencing-server/test/seqjson-to-edsl.spec.ts

```diff
@@ -54,8 +55,49 @@ describe('getEdslForSeqJson', () => {
   });`,
     );
   });
+    */
+
+  it('should throw an error if the user uploads an invalid seqjson', async () => {
+    try {
```

Comment:
You shouldn't need the try-catch anymore, and I suspect you may need to await the expect() call to ensure the test runner waits for the expectation to resolve.

100% spitballing based on experience and not running these tests though, so if you're seeing otherwise, that's great!

Reply:
Yeah, I tried removing the try-catch but it still threw the error, so I just left it to keep the testing logs clean!

Reply:
I suspect that means the test is not setup correctly.

Try this
```typescript
it('should throw an error if the user uploads an invalid seqjson', async () => {
 await expect(
   graphqlClient.request<{
     getEdslForSeqJson: string;
   }>(
     gql`
       query GetEdslForSeqJson($seqJson: SequenceSeqJsonInput!) {
         getEdslForSeqJson(seqJson: $seqJson)
       }
     `,
     {
       seqJson: {
         id: 'test_00001',
         metadata: {},
         steps: [
           {
             // expansion 1
             type: 'command',
             stem: 'BAKE_BREAD',
             time: { type: TimingTypes.COMMAND_COMPLETE },
             args: [],
             metadata: {},
           },
           {
             type: 'command',
             stem: 'PREHEAT_OVEN',
             time: { type: TimingTypes.ABSOLUTE, tag: '2020-060T03:45:19.000Z' },
             args: [100],
             metadata: {},
           },
         ],
       },
     },
   ),
 ).rejects.toThrow();
});
```